config/jobs/kubernetes-sigs/kueue/kueue-release-blocking-main.yaml

```diff
@@ -437,3 +437,49 @@ periodics:
             limits:
               cpu: "7"
               memory: "10Gi"
+  - interval: 12h
+    name: periodic-kueue-test-e2e-certmanager-main
+    cluster: eks-prow-build-cluster
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-test-e2e-certmanager-main
+      testgrid-alert-email: kueue-alerts@kubernetes.io
+      testgrid-num-failures-to-alert: '1'
+      description: "Run periodic certmanager end to end tests"
+      testgrid-num-columns-recent: '30'
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: main
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250227-3a13bdd784-master
+          env:
+            - name: E2E_KIND_VERSION
+              value: kindest/node:v1.31.1
```

Comment:
same here.

Reply:
I think 1.31.1 is fine for this PR for consistency with other CI jobs, like `pull-kueue-test-e2e-tas-main`. I think we can have an orthogonal follow up bumping to 1.32.x (basically latest)